Adding a new WildFly Bootable JAR on OpenShift test

fabiobrz opened the pull request:
#### Short description of what this resolves:

SSIA

#### Changes proposed in this pull request:

- A new functional test to verify how a modern WildFly Bootable JAR can be deployed and tested on OpenShift 
- A fix to the WildFly Bootable JAR k8s test artifact id

Fixes #1374

gaol commented on openshift/ftest-openshift-wildfly-bootable-jar/src/test/java/org/arquillian/cube/kubernetes/WildflyBootableJarOpenShiftIT.java:
```diff
@@ -0,0 +1,75 @@
+    @InSequence(2)
+    public void helloEndpointShouldReplyWithHttp200() throws InterruptedException {
+        // wait until one route is actually ready, 60 seconds at most currently works for CI...
+        Thread.sleep(60 * 1000);
```
Maybe it would be better to use the `Awaitility.await()` API to check on conditions until 60 seconds timeout instead of sleeping for 60 seconds before the check.

gaol reviewed (approved):
LGTM with trivial comments. :)

fabiobrz commented:
> LGTM with trivial comments. :) 

Hi @gaol, I addressed your comments, and BTW added a couple of commits to fix two tests that started failing during the last two days, most likely due to GutHub runners specifics around networking. Not a big deal but feel free to check and let me know what you think. 
CI checks are all green now! 🙂
